init.lua

```diff
@@ -363,4 +366,10 @@ function throwing.register_bow(name, def)
 		return itemstack
 	end
 	minetest.register_tool(name, def)
+	if use_toolranks then
+		minetest.override_item(name:sub(2), {
```

Comment:
This assumes that the item name as passed to the function always starts with a `:` prefix, which is not necessarily true. A check should be made to remove the prefix if it exists, but keep the whole name otherwise.

Reply:
Oh, you are right, fixed it.